docs/MobileStepper.md

```diff
@@ -18,20 +18,30 @@ imports: [
 ```
 
 ```html
-// your-component.ts
-activeStep = 0;
-steps = 4;
+your-component.ts activeStep = 0; steps = 4;
 ```
 
 ```html
-// your-component.html
+your-component.html
```

Comment:
this needs to be commented

Reply:
not sure why this MobileStepper file is having so many problems. The UserMenu has a similarly commented snippet using the `typescript` designation and it seems to be working fine.